bundles/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComMessageFactory.java

```diff
@@ -43,7 +43,14 @@
                     put(PacketType.LIGHTING5, RFXComLighting5Message.class);
                     put(PacketType.LIGHTING6, RFXComLighting6Message.class);
                     put(PacketType.CHIME, RFXComChimeMessage.class);
-                    // put(PacketType.FAN, RFXComFanMessage.class);
+                    put(PacketType.FAN, RFXComFanMessage.class);
+                    // put(PacketType.FAN_SF01, RFXComFanMessage.class);
```

Comment:
Why are these commented? Can they be removed?

Reply:
We have these entries also for other cases which are not yet supported, so I followed that existing pattern